pkg/integration/tests/misc/recent_repos_on_launch.go

```diff
@@ -0,0 +1,28 @@
+package misc
+
+import (
+	"github.com/jesseduffield/lazygit/pkg/config"
+	. "github.com/jesseduffield/lazygit/pkg/integration/components"
+)
+
+// Couldn't find an easy way to actually reproduce the situation of opening outside a repo,
+// so I'm introducing a hacky env var to force lazygit to show the recent repos meu upon opening.
```

Comment:
Some more context: if we delete the .git directory, lazygit just travels up the directory chain to the root looking for a repo, and it _finds_ one because this all sits inside the lazygit repo. But there's also logic to just fail if you pass the --path arg directly and the repo doesn't exist (which we do with integration tests). So it's just tricky to test this thing faithfully